bodo/pandas/series.py

```diff
@@ -18,6 +21,37 @@ class BodoSeries(pd.Series, BodoLazyWrapper):
     _head_s: pd.Series | None = None
     _name: Hashable = None
 
+    def __init__(
+        self,
+        data=None,
+        index=None,
+        dtype: Dtype | None = None,
+        name=None,
+        copy: bool | None = None,
+        fastpath: bool | lib.NoDefault = lib.no_default,
+        plan: plan_operators.PlanOperator | None = None,
+    ) -> None:
+        self.__dict__["plan"] = plan
```

Comment:
Using `__dict__` doesn't seem necessary since "plan" is not a dynamic value?

Reply:
Pandas overloads __getattr__ and that function doesn't recognize "plan" as being valid.  You'll get an error if you don't set it this way.  We could also do our own overload of __getattr__ in BodoDataFrame.

Reply:
Let's add a comment to explain this since not trivial.

Reply:
Fixed using the proper pandas mechanism.